packages/node/package.json

```diff
@@ -14,13 +14,12 @@
     "lint": "eslint \"src/**/*.ts\"",
     "lint-fix": "eslint --fix \"src/**\"",
     "licenses": "license-checker --production --out license.csv --failOn \"AGPL-1.0-only; AGPL-1.0-or-later; AGPL-3.0-only; AGPL-3.0-or-later; Beerware; CC-BY-NC-1.0; CC-BY-NC-2.0; CC-BY-NC-2.5; CC-BY-NC-3.0; CC-BY-NC-4.0; CC-BY-NC-ND-1.0; CC-BY-NC-ND-2.0; CC-BY-NC-ND-2.5; CC-BY-NC-ND-3.0; CC-BY-NC-ND-4.0; CC-BY-NC-SA-1.0; CC-BY-NC-SA-2.0; CC-BY-NC-SA-2.5; CC-BY-NC-SA-3.0; CC-BY-NC-SA-4.0; CPAL-1.0; EUPL-1.0; EUPL-1.1; EUPL-1.1;  GPL-1.0-only; GPL-1.0-or-later; GPL-2.0-only;  GPL-2.0-or-later; GPL-3.0; GPL-3.0-only; GPL-3.0-or-later; SISSL;  SISSL-1.2; WTFPL\"",
-    "test": "npm run lint-fix && npm run licenses && jest --coverage --verbose --passWithNoTests",
+    "test": "npm run lint-fix && npm run licenses && jest --coverage --verbose --passWithNoTests --no-cache",
```

Comment:
Was this just a temporary thing, or is there a genuine reason for turning off the cache (according to the Jest docs: "**_On average, disabling the cache makes Jest at least two times slower_**")...?

Reply:
Yup, that's an oversight, my mistake. Fixed in b556401d2d5c01986f1c6f1f2718b38827ea813b